### ISSUE TITLE
Error al cancelar

### ISSUE THREAD

**Issue by mvr1981** [gpg4usb/gpg4usb#30]
gpg4usb 0.3.3

Al intentar decifrar un fichero el programa pide el ingreso de la contraseña correspondiente: en Windows XP (sin privilegios de administrador) el programa se cuelga (application crashes) cuando se cancela sin ingresar la contraseña correspondiente.

When trying to decipher a file the program requests the entry of the password: in Windows XP (without administrative privileges), the program hangs (application crashes) when canceled without entering the password.

Thanks...

**Comment by vatterspun** [comment]
Yeah it should definitely not do that.  This program has no reason to ask for admin privileges.  Do you happen to have a screenshot?
